uint8 FREE = 0
uint8 RESERVED = 1
uint8 BUSY = 2
uint8 UNAVAILABLE = 3
uint8 ERROR = 4

Header header
uint8 state